src/modules/Reviews.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * Reviews
+ */
+namespace WPML\PB\Elementor\Modules;
+
+class Reviews extends \WPML_Elementor_Module_With_Items {
+
+    protected function get_title( $field )
+    {
+        switch ( $field ) {
+            case 'content':
+                return esc_html__( 'Comment Contents', 'sitepress' );
+            case 'name':
+                return esc_html__( 'Commenter Name', 'sitepress' );
+            case 'title':
+                return esc_html__( 'Comment Title', 'sitepress' );
+            case 'image':
```

Comment:
Images are handled by Media Translation?

Reply:
Yes, they should be.
However, we already have the exception case here: https://github.com/OnTheGoSystems/wpml-page-builders-elementor/pull/114 with self-hosted videos.
That is why I was trying to make it work in the same fashion.
Perhaps @strategio can give advice on what to do here.

Reply:
I looked further at this.
It is a different case than the self-hosted video.
In the self-hosted video you can specify URL, while here you can only use the media gallery to set the attachment.
I am going to remove the ability to manually specify the image path because of that.

Reply:
If we want to translate the image, we need to create a new media module in `src/media/modules` with a class extending `\WPML_Elementor_Media_Node`.

I am sharing for you to know, but I would not bother implement this here. The image should be a picture of the reviewer, so I don't see why it would be converted.